### ISSUE TITLE
Incorrect message output to logs

### ISSUE THREAD

**Issue by sharpchi** [turnitin/moodle-mod_turnitintooltwo#477]
In the activity logs when a Turnitin activity is deleted a "General Clean Up" is run (turnitintooltwo_assignment.class.php)
It iterates over all the courses in the turnitintooltwo_courses table, and checks to see if a. the course still exists in Moodle, and b. if it exists in the turnitintooltwo table.
In both cases, whether it actually deletes anything or not, the activitylog declares that the "Old Moodle Course" has been deleted. It gave me the willies seeing that. The message only needs to be displayed if it has deleted something.
```// General Clean Up.
        if ($oldcourses = $DB->get_records("turnitintooltwo_courses")) {
            foreach ($oldcourses as $oldcourse) {
                // Delete the Turnitin Classes data if the Moodle courses no longer exists.
                if (!$DB->count_records("course", array("id" => $oldcourse->courseid)) > 0) {
                    $DB->delete_records("turnitintooltwo_courses", array("courseid" => $oldcourse->courseid));
                }
                // Delete the Turnitin Class data if no more turnitin assignments exist in it.
                if (!$DB->count_records("turnitintooltwo", array("course" => $oldcourse->courseid)) > 0) {
                    $DB->delete_records("turnitintooltwo_courses", array("courseid" => $oldcourse->courseid,
                                                        "course_type" => "TT"));
                }
    -->        turnitintooltwo_activitylog("Old Moodle Course deleted - id (".$oldcourse->courseid." - ".
                                                        $oldcourse->turnitin_cid.")", "REQUEST");
            }
        }
```

**Comment by jmcgettrick** [comment]
Hey @sharpchi - yes that's probably a bit of a scare if you see that. We'll get it changed.

**Comment by jmcgettrick** [comment]
Fixed by #610
